typescript/src/ClientCredentialsGrantAuthenticator.ts

```diff
@@ -1,6 +1,6 @@
 import getTokenWithClientCredentialsGrant from "./getTokenWithClientCredentialsGrant";
 
-export default class ClientCredentialsGrantAuthenticator {
+export class ClientCredentialsGrantAuthenticator {
```

Comment:
@secana so when having an 

`export default'`

you don't need the

`"allowSyntheticDefaultImports":` true `

flag right? Is there any reason to remove the default?